### ISSUE TITLE
SQL Server ODBC driver upgrade

### ISSUE THREAD

**Issue by sirio3mil** [Andrewpk/Microsoft--SQL-Server--ODBC-Driver-1.0-for-Linux-Fixed-Install-Scripts#12]
Microsoft has release version 13 (preview) for its ODBC driver, compatible with linux 7 ¿changing names in your script it will works?

**Comment by r2evans** [comment]
On an initial scrub, I tried "changing the names" on his script and it did not work. This was a cursory scan, admittedly, so take that with a grain of salt. I'm currently using MSODBC 11 with unixODBC-2.3.2 and it works fine, so I'm loath to switch without reason and testing.

The only "changes" in MSODBC 13 I found so far are [here](https://msdn.microsoft.com/en-us/library/jj730313%28v=sql.110%29.aspx), which highlights "Always Encrypted" as the only difference from 11 to 13. Though I find this hard to believe (at least "stability, performance improvements, bug fixes"), my use-case doesn't demand this feature so it may be moot for me.
